src/ralph/domains/admin.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin import RalphAdmin, RalphTabularInline, register
+from ralph.domains.models.domains import (
+    Domain,
+    DomainContract,
+    DomainRegistrant
+)
+
+
+class DomainContractInline(RalphTabularInline):
+    model = DomainContract
+    extra = 0
+
+
+@register(Domain)
+class DomainAdmin(RalphAdmin):
```

Comment:
don't we want to use attachments here? if yes, mix `AttachmentsMixin` (`ralph.attachments.admin.AttachmentsMixin`)